packages/myst-cli/src/project/load.ts

```diff
@@ -57,9 +57,13 @@ export async function loadProjectFromDisk(
   let newProject: Omit<LocalProject, 'bibliography'> | undefined;
   let { index, writeTOC } = opts || {};
   let legacyToc = false;
+  const siteConfig = selectors.selectLocalSiteConfig(state, path);
+  const urlFolders = !!siteConfig?.options?.url_folders;
```

Comment:
I'm happy to take suggestions about a better name for this option...

Reply:
@fwkoch do we have a policy about `snake_case` vs `pascalCase` for frontmatter/config item names?

Reply:
lets do `site.folders` and get around it!